src/components/layout/CustomIntercom.js

```diff
@@ -19,7 +18,8 @@ import { Badge, IconButton, Tooltip, useTheme } from "@material-ui/core";
 import LiveHelpIcon from "@material-ui/icons/LiveHelp";
 
 function CustomIntercom(props) {
-    const { intercomUnreadCount, t } = props;
+    const { intercomUnreadCount } = props;
+    const { t } = useTranslation("common");
```

Comment:
Just picking a random spot to comment because the quoting thing is harder to follow - for an example of issues we've had with `react-intl` and having 2 components working together with 2 different messages files: `withErrorAnnouncer`.  The error announcer is using the `utils` directory messages file, and any component that uses `withErrorAnnouncer` will be using its own messages file.  `getMessage` for some reason didn't work (when using the `Details` button, the details text could never be found, IIRC), but `formatMessage` did work.

Reply:
Let me check this.